BioCrypt-Personal-V0.4 Source-code/main.py

```
@@ -0,0 +1,1814 @@
+        # Pad the plaintext to match the block size
+        padded_plaintext = pad(plaintext, AES.block_size)
+
+        # Encrypt the padded plaintext
+        ciphertext = cipher.encrypt(padded_plaintext)
+
+        # Save the ciphertext to the encrypted file
+        encrypted_file_path = file_path + ".enc"
+        with open(encrypted_file_path, "wb") as f:
+            f.write(ciphertext)
+
+        return f"File encrypted successfully: {encrypted_file_path}"
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)